dist/w2ui-dark.css

```diff
@@ -1,7 +1,7 @@
 /* w2ui 1.5.x (nightly) (c) http://w2ui.com, vitmalina@gmail.com */
```

Comment:
all files in /dist/* folder are compiled. No need to check them into repository.